jenkins/debian/debos/scripts/crush.sh

```diff
@@ -10,10 +10,9 @@ UNNEEDED_PACKAGES="apt libapt-pkg5.0 "\
 "ncurses-bin ncurses-base libncursesw5 libncurses5 "\
 "perl-base "\
 "debconf libdebconfclient0 "\
-"e2fsprogs e2fslibs libfdisk1 libdevmapper1.02.1 "\
```

Comment:
On the other hand, these things aren't needed in most cases.  So it would be good to have a way to keep them in the `stretch-ltp` file system but not in others.

It's also worth checking the difference in the image size this adds, because if it's negligible then it's not so much of an issue.